src/scipp/plotting/__init__.py

```diff
@@ -86,6 +86,11 @@ def plot(*args, **kwargs):
         is used. `labels={"time": "time-labels"}`. Defaults to `None`.
     :type labels: dict, optional
 
+    :param camera: Define initial camera 'position' and 'look_at' (3d only).
+        Defaults to `None`. When nothing is specified, the camera looks at the
+        mean position of the pixels.
```

Comment:
I do not understand how the default setttings look like: "looks at mean position of the pixels"

Reply:
'The barycentric position of the pixels'?

Reply:
What was unclear for me was the plural: pixels?
I thought that the camera was pointing at one place only.
So maybe "...the camera looks at the barycentric position of the targeted pixel"

Reply:
Not sure I understand what you mean. There is no targeted pixel. Maybe the confusion is because I used the word 'pixel', which is probably not correct. I was thinking of 'detector pixel', but the plotting in scipp is actually non neutron-specific.

There is a cloud of points, and by default the camera will look at the center of the cloud, which is what i was trying to say with 'barycentric position'.

Should I change to 'When nothing is specified, the camera looks at the center of the cloud of plotted points.'

Reply:
👍 for the last suggestion.

Reply:
I agree with Simon.
I had completely misunderstood the first versions.